src/coreclr/ToolBox/superpmi/superpmi/parallelsuperpmi.cpp

```diff
@@ -539,6 +539,12 @@ int doParallelSuperPMI(CommandLine::Options& o)
                                       arrDiffMCListPath[i]);
         }
 
+        if (o.failureLimit > 0)
+        {
+            bytesWritten += sprintf_s(cmdLine + bytesWritten, MAX_CMDLINE_SIZE - bytesWritten, " -failureLimit %d",
+                                      o.failureLimit);
```

Comment:
This is fine. Note that there's a slight semantic difference here, in that this will be a failure limit *per parallel process*, not a failure limit overall. Given that the goal is to avoid going crazy with what will be unused failure information, that's fine.

Reply:
Yes, that's true.